runners/portability/java/build.gradle

```
@@ -32,12 +32,19 @@ configurations {
 
 dependencies {
   compile library.java.vendored_guava_26_0_jre
-  compile library.java.hamcrest_library
   compile project(":runners:java-fn-execution")
   compile project(":runners:java-job-service")
   compile project(path: ":sdks:java:harness", configuration: "shadow")
   compile library.java.vendored_grpc_1_26_0
   compile library.java.slf4j_api
+  compile library.java.joda_time
+  compile "org.hamcrest:hamcrest:2.1"
+  compile project(path: ":model:fn-execution", configuration: "shadow")
+  compile project(path: ":model:job-management:", configuration: "shadow")
+  compile project(path: ":model:pipeline", configuration: "shadow")
+  compile project(":runners:core-construction-java")
+  permitUsedUndeclared project(":sdks:java:core")
```

I'm getting that same error you got if I remove this line:

```
> No signature of method: ca.cutterslade.gradle.analyze.AnalyzeDependenciesTask$_action_closure3$_closure8.doCall() is applicable for argument types: (org.gradle.api.internal.artifacts.PreResolvedResolvableArtifact) values: [beam-sdks-java-core.jar (project :sdks:java:core)]
  Possible solutions: findAll(), findAll(), isCase(java.lang.Object), isCase(java.lang.Object)
```

I wonder if the plugin has a bug with handling diamond dependencies.

---

I too wonder on this error as I used to get error differently before. Is there anything related to this that has being updated/integrated recently?

---

Since, I have rebased with master, i have been getting: 
```
No signature of method: ca.cutterslade.gradle.analyze.AnalyzeDependenciesTask$_action_closure3$_closure8.doCall() is applicable for argument types: (org.gradle.api.internal.artifacts.PreResolvedResolvableArtifact) values: [beam-sdks-java-core.jar (project :sdks:java:core)]
  Possible solutions: findAll(), findAll(), isCase(java.lang.Object), isCase(java.lang.Object) 
```
Else it works fine as before.

---

https://github.com/wfhartford/gradle-dependency-analyze/issues/125

---

Anyway, why can't we just declare this one?

```suggestion
  compile project(path: ":sdks:java:core", configuration: "shadow")
```